docs/develop/customize.md

```diff
@@ -22,3 +24,16 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## Change colors
+
+Another common request that you might have is: *How do I change the colors so I can make my instance look like my institution's theme?*. We show what changes you need to do in order to achieve that.
+
+We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/scss/<your instance shortname>/variables.scss` file and edit it as below:
+
+``` console
+$navbar_background_image: unset;
+$navbar_background_color: #myColorCode;
```

Comment:
Minor: I would put an example, experience thought us when you put text or similar user might think that variable is somewhere to be set.
```suggestion
$navbar_background_color: #FFFFFF;  // Note this hex value is an example. Choose yours.
```